### PR TITLE
Kraken cleanup and fix

### PR DESCRIPTION
**Pull request by davebx**

### CODE REVIEW

**Comment by bgruening** [comment]
Uh this seems to be a serious bug. Sorry for not catching it before.
Is it possible to get a test for this tool?

**Comment by davebx** [comment]
@bgruening Yeah, I'll get something worked up.

**Comment by davebx** [comment]
@bgruening Oh wait, kraken is challenging to test, due to the need for a rather large database. Let me give it some thought.

File: tool_collections/kraken/kraken_report/kraken-mpa-report.xml [hunk 1]
```diff
@@ -1,38 +1,35 @@
...
+<tool id="kraken-mpa-report" name="Kraken-mpa-report" version="1.1.1">
+    <description>view report of classification for multiple samples</description>
     <macros>
         <import>macros.xml</import>
     </macros>
     <command>
-      <![CDATA[
+        <expand macro="requirements" />
+        <expand macro="stdio" />
+        <expand macro="version_command" />
```
Comment (nsoranzo) [inline tool_collections/kraken/kraken_report/kraken-mpa-report.xml]: The 3 macros should be outside of `<command>`.

**Comment by martenson** [comment]
:+1:

**Pull request merged**